docs/index.html

```diff
@@ -34,6 +34,32 @@
 			background-repeat: no-repeat;
 		"
 	>
+
+		<!-- Script for scrolling-->
+		<script>		
+
+			window.addEventListener('scroll', function() {
```

Comment:
Highly inefficient code... searching for element IDs and calculating bounds _everytime_ the page is scrolled.

Reply:
You are doing a .getElementByID **every** single time the page scrolls, pretty sure thats what they are referencing.